static/templates/compose_banner/compose_banner.hbs

```diff
@@ -1,5 +1,5 @@
 <div
-  class="compose_banner {{banner_type}} {{classname}}"
+  class="compose_banner composebox_banner {{banner_type}} {{classname}}"
```

Comment:
What's the motivation for not removing the `compose_banner` class here -- is that still used for something?

Reply:
It's used for things like

```
    const $compose_banner_area = $("#compose_banners");
    $compose_banner_area.append(new_row);
```

And the success banner is a different container than that.